src/app/pages/listing/listing.jsx

```diff
@@ -361,7 +365,7 @@ function ChplListingPage({ id, panel }) {
                    <span id="sed" className="page-anchor" />
                  </span>
                  <Box className={classes.sectionHeader}>
-                   <Typography variant="h2">Safety Enhanced Design (SED)</Typography>
+                   <Typography className={classes.sectionHeaderText} variant="h2">Safety Enhanced Design (SED)</Typography>
```

Comment:
The first few headers have this line all on one line, and the last few have it broken into three. I don't care too much which way we go, but it should be consistent